uniffi_bindgen/src/templates/macros.rs

```diff
@@ -65,12 +65,12 @@ use uniffi::UniffiMethodCall;
 {%- endif -%}
 {% match meth.throws() -%}
 {% when Some with (e) -%}
-{{ this_handle_map }}.method_call_with_result(err, {{ meth.first_argument().name() }}, |obj| -> Result<{% call return_type_func(meth) %}, {{e}}> {
+{{ this_handle_map }}.method_call_{% if meth.takes_self_by_arc() %}by_arc_{% endif %}with_result(err, {{ meth.first_argument().name() }}, |obj| -> Result<{% call return_type_func(meth) %}, {{e}}> {
```

Comment:
🤮 at my own work here, but if it's useful to get this in early, I think we can tolerate this for a short while.

Reply:
heh